iothub/service/src/Common/Amqp/ClientWebSocketTransport.cs

```diff
@@ -58,7 +58,7 @@ public override bool WriteAsync(TransportAsyncCallbackArgs args)
             Fx.AssertAndThrow(args.CompletedCallback != null, "must have a valid callback");
             args.Exception = null; // null out any exceptions
 
-            Task taskResult = WriteAsyncCore(args);
+            Task taskResult = WriteAsyncCoreAsync(args);
```

Comment:
Just WriteAsync?

Reply:
there is already WriteAsync..maybe WriteCoreAsync